src/styles/blocks/top-row-layout.scss

```diff
@@ -0,0 +1,33 @@
+.top-row-layout {
+
+  &--one-icon {
+    @include headerNavigationGrid(1, 18px, 30px);
+    @include onTablet {
+      @include headerNavigationGrid(1, 24px, 63px);
+    }
+  }
+
+  &--two-icons {
+    @include headerNavigationGrid(2, 18px, 30px);
+    @include onTablet {
+      @include headerNavigationGrid(2, 24px, 63px);
+    }
+  }
+
+  &__link-logo {
+    cursor: auto;
+  }
+
+  &__logo {
+    cursor: pointer;
```

Comment:
The 'cursor: pointer;' property should be applied to the '.top-row-layout__link-logo' element, not the '.top-row-layout__logo' element. This ensures that the cursor changes to a pointer when hovering over the link, not the logo image itself.